UnityProject/Assets/Scripts/Objects/Gateway/TransportUtility.cs

```diff
@@ -71,22 +71,13 @@ public static void TransportObject(UniversalObjectPhysics objectPhysics, Vector3
 			for (var node = linkedList.First; node != null; node = node.Next)
 			{
 				var currentObj = node.Value;
-				var previous = node.Previous?.Value;
-
-				//Disconnect pulling to make it not be a problem
-				currentObj.PullSet(null, false); //TODO Test without
 
 				//Transport current
 				TransportObject(currentObj, transportTo, doTileStep, maintRoomChanceModifier);
 
-				if (previous != null && currentObj.gameObject != null)
-				{
-					//There was another object before this one, pulling it. Re-establish pulling. (But only if the current object's gameObject is not null)
-					previous.CmdPullObject(currentObj.gameObject);
-				}
-
 				//TODO: Make pulling acutally continue working across teleporters for clients, not just server
 				//TODO: Find a way to make the teleporter the teleport not all be on the same tile.
```

Comment:
when A and B teleport and A is pulling B, both of them occupy the same tile after teleporting.

Reply:
...but that's exactly what happens now?

Reply:
nevermind, i double checked; it doesn't.
though i do have to ask, why does it matter? ss13 has similar behavior to what I've made in this PR, and it seems to be fine?

Reply:
I'm merely explaining  Bod's comments. I do think it is nice not to break the conga line when teleporting